src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1519,4 +1519,13 @@
     <value>and all other platforms</value>
     <comment>This call site is reachable on: 'windows' 10.0.2000 and later, and all other platforms</comment>
   </data>
+  <data name="ProvidePublicParameterlessSafeHandleConstructorDescription" xml:space="preserve">
+    <value>Providing a public parameterless constructor for a type derived from System.Runtime.InteropServices.SafeHandle enables better performance and usage with source-generated interop solutions.</value>
+  </data>
+  <data name="ProvidePublicParameterlessSafeHandleConstructorMessage" xml:space="preserve">
+    <value>Provide a public parameterless constructor for concrete types derived from System.Runtime.InteropServices.SafeHandle</value>
+  </data>
+  <data name="ProvidePublicParameterlessSafeHandleConstructorTitle" xml:space="preserve">
+    <value>Provide a public parameterless constructor for the '{0}' type that is derived from System.Runtime.InteropServices.SafeHandle</value>
```

Comment:
Titles cannot have place holders, only messages do.
Also we surround type names with single quotes (applies to other resources).